lib/vagrant-service-manager/command.rb

```diff
@@ -5,6 +5,7 @@ module Vagrant
   module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
+    SCCLI_SERVICES = ["openshift", "kubernetes", "k8s"]
```

Comment:
Looking at the time constraint here, my comment should not stop you from merging the patch. We can re-factor the code post release.